packages/fetch-engine/src/__tests__/download.test.ts

```diff
@@ -450,20 +531,34 @@ Array [
         'darwin-arm64',
         'debian-openssl-1.0.x',
         'debian-openssl-1.1.x',
+        'debian-openssl-3.0.x',
         'linux-arm64-openssl-1.0.x',
         'linux-arm64-openssl-1.1.x',
+        'linux-arm64-openssl-3.0.x',
         'rhel-openssl-1.0.x',
         'rhel-openssl-1.1.x',
+        'rhel-openssl-3.0.x',
         'windows',
         'linux-musl',
       ],
       version: FIXED_BINARIES_HASH,
     })
     const after2 = Date.now()
-    // value on Mac: 33ms
-    // value on GH Actions: ?
-    // https://github.com/prisma/prisma/runs/1176632754
-    const took2 = after2 - before2
-    expect(took2).toBeLessThan(10000)
+    const timeInMsToDownloadAllFromCache2 = after2 - before2
+    console.debug(
+      `3 - With cache2: Engines were already present
+It took ${timeInMsToDownloadAllFromCache2}ms to execute download() for all binaryTargets.`,
+    )
+
+    // This is a rather high number to avoid flakiness in CI
+    expect(timeInMsToDownloadAll).toBeLessThan(20_000)
```

Comment:
I believe it only used to check what's now `timeInMsToDownloadAllFromCache1` and `timeInMsToDownloadAllFromCache2`.

20000 is not enough for downloading (and there probably isn't any reasonable number to write a useful and non-flaky test for `timeInMsToDownloadAll`):

<img width="661" alt="image" src="https://user-images.githubusercontent.com/4923335/164088549-339e0940-2416-468b-a293-1fe46ddf4d66.png">

Reply:
```suggestion
    expect(timeInMsToDownloadAll).toBeLessThan(80_000)
```

Reply:
I'd rather remove this assertion completely. There's no property to test here, downloading may take unpredictable amount of time, and it doesn't depend on us or our code. Testing the time after caching still make sense, of course.